modules/affile/affile-source.c

```diff
@@ -178,5 +178,6 @@ affile_sd_new(gchar *filename, GlobalConfig *cfg)
 
   self->file_reader_options.restore_state = self->file_reader_options.follow_freq > 0;
   file_opener_options_defaults_dont_change_permissions(&self->file_opener_options);
+  self->file_opener_options.create_dirs = FALSE;
```

Comment:
> We need to explicitly disable directory creation, as a global create_dirs() option with a value "yes" can enable it and we don't want that.

As a default, I agree; we should disable the inheritance. Creating the directory structure (not the file itself) for the monitored file is not necessarily a bad idea, if someone explicitly sets the option in the driver.

Reply:
I noticed we've never had such an option for the file source. We can add it later, if someone needs it.